test/output/plot_output.R

```diff
@@ -1,11 +1,22 @@
 library(tidyverse)
```

Comment:
I'm assuming you don't want to actually commit `Rplots.pdf`? Or was that intentional